static/src/stylesheets/module/commercial/creatives/_fabric-video.scss

```diff
@@ -19,6 +19,9 @@
 
     .creative__video {
         object-fit: cover;
+
+        // this font-family style is for use with the 'object-fit-videos' polyfill.
+        font-family: 'object-fit: cover;';
```

Comment:
eurgh! does width/height = 100% not work?

Reply:
This hack is used by the polyfill module. The intersection of 'polyfill that works' and 'polyfill with reasonable modular structure' is very small.